src/wallet/wallet.cpp

```diff
@@ -3780,7 +3780,7 @@ std::unique_ptr<WalletDatabase> MakeWalletDatabase(const std::string& name, cons
     // 2. Path to an existing directory.
     // 3. Path to a symlink to a directory.
     // 4. For backwards compatibility, the name of a data file in -walletdir.
-    const fs::path& wallet_path = fs::absolute(name, GetWalletDir());
+    const fs::path& wallet_path = fsbridge::AbsPathJoin(GetWalletDir(), name);
```

Comment:
```suggestion
    const fs::path wallet_path = fsbridge::AbsPathJoin(GetWalletDir(), name);
```

Reply:
I understand that one is supposed to apply the commit locally without using Github's "Commit suggestion" as that would lead to a new CI run, right?

Reply:
@kiminuo Yes, I don't recall seeing someone use the GitHub web "commit suggestion" interface here other than for just giving feedback, but you may be right. It's probably best (and more secure) to use git locally and not become dependent on GitHub-specific workflows.